pathoscore.py

```diff
@@ -549,6 +597,31 @@ def step_plot(vals, ax, **kwargs):
     ax.plot(p_edges, p, ls='steps', lw=1.9, **kwargs)
     return p_edges, p
 
+def plotly_html(score_methods, score_counts, roc_traces, jbar_trace, jdist_traces, score_step_divs, step_traces, scorable, prefix, cu=[], header=[]):
```

Comment:
let's leave this for now, but after this PR, let's make this function take a dictionary. otherwise it's too error-prone.

Reply:
understood.